python/src/viewer.py

```diff
@@ -576,32 +632,70 @@ def save(self, fname, **kwargs):
             A string containing a path to a filename from which file format is deduced.
 
         kwargs:
-            Refer to matplotlib.figure.Figure.savefig documentation for valid keyword arguments.
+            See https://matplotlib.org/api/_as_gen/matplotlib.figure.Figure.html#matplotlib.figure.Figure.savefig documentation for valid keyword arguments.
```

Comment:
Same remark as previously

Reply:
Ok. I remove all links to the Matplotlib website.